MvvmCross/Platforms/Android/Binding/ResourceHelpers/MvxAppResourceTypeFinder.cs

```diff
@@ -1,22 +1,35 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MS-PL license.
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Reflection;
 using MvvmCross.Exceptions;
 
 namespace MvvmCross.Platforms.Android.Binding.ResourceHelpers
 {
     public class MvxAppResourceTypeFinder : IMvxAppResourceTypeFinder
     {
+        private Type FindResourceType(Assembly assembly)
+        {
+            var att = assembly.CustomAttributes.FirstOrDefault(ca =>
+                          ca.AttributeType.FullName == "Android.Runtime.ResourceDesignerAttribute" &&
+                          ca.Properties.First(p => p.Name == "IsApplication").Argument.Value == true);
```

Comment:
`ca.Properties` doesn't seem to be present.

Reply:
heh - this is Mono.Cecil API, not System.Reflection 🤪

Reply:
Should still be fine to look for `ResourceDesignerAttribute`. However, instead of checking for `IsApplication` maybe just assume that the assembly with `Setup` file, is the Application.

Reply:
OK. That makes it easier. I know nothing about MvvmCross by the way - never used it. My C# is a little rusty too :)

Reply:
![image](https://user-images.githubusercontent.com/667194/38491258-44d444ee-3be3-11e8-9550-412db5d1e98f.png)
Let me get this into the PR. Will rebase.